htdocs/wp-content/themes/hollandskroon/src/Commands/PronamicIdealCommand.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace HK\Commands;
+
+use \WP_CLI;
+use \WP_CLI\ExitException;
+
+class PronamicIdealCommand
+{
+	/**
+	 * @throws \JsonException
+	 * @throws ExitException
+	 */
+	public function __invoke($args, $assoc_args)
+	{
+		$paged = 1;
+		$posts_per_page = 100;
+
+		do {
+			// Get fulfilled payments older than 5 days in batches of 100
+			$payments = new \WP_Query(array(
+				'post_type'      => 'pronamic_payment',
+				'post_status'    => 'payment_completed',
+				'posts_per_page' => $posts_per_page,
+				'paged'          => $paged,
+				'date_query'     => array(
+					'before' => '5 days ago',
+				),
+			));
+
+			if (!$payments->have_posts()) {
+				WP_CLI::error('No fulfilled payments found to clean up.');
+			}
+
+			foreach ($payments->posts as $payment) {
+				if (!wp_delete_post($payment->ID, true)) {
+					WP_CLI::error('Failed to delete post with ID: ' . $payment->ID);
```

Comment:
https://make.wordpress.org/cli/handbook/references/internal-api/wp-cli-error/

"Defaults to halting script execution", is dit wat je wilt? Wellicht is [deze](https://make.wordpress.org/cli/handbook/references/internal-api/wp-cli-warning/) meer op zijn plaats?